TestResultSummaryService/parsers/BenchmarkMetricRouter.js

```diff
@@ -1,32 +1,113 @@
     "ILOG_WODM": {
         "881-4way-Seg5FastpathRVEJB": "ILOG_WODM",
-        "881-4way-Seg300RulesFastpathRVEJB": "ILOG_WODM"
+        "881-4way-Seg300RulesFastpathRVEJB": "ILOG_WODM",
+        "4way-Seg5FastpathRVEJB": "ILOG_WODM",
+        "4way-Seg300RulesFastpathRVEJB": "ILOG_WODM"
     },
     "SPECjbb2015": {
         "multi_2grp_gencon": "SPECjbb2015"
     },
+    "Crypto-BB":{
+        "default": "Crypto-BB",
+    },
+// ##################################### PERFFARM SPECIFIC ROUTER VALUES ################################
+   "CleanedLibertyStartup": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartupDT": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-Xint-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartupDT7": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-Xint-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartupHugeEJB": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyThroughput-DayTrader3":{
+        "9dev-4way-LargeThreadPool": "LibertyDTThroughput",
+        "9dev-4way-LargeThreadPoolwarm": "LibertyDTThroughput",
+        "17dev-4way-LargeThreadPool": "LibertyDTThroughput",
+        "17dev-4way-LargeThreadPoolwarm": "LibertyDTThroughput"
+    },
+    "CleanedLibertyThroughput-DayTrader7":{
+        "9dev-4way-LargeThreadPool": "LibertyDTThroughput",
+        "9dev-4way-LargeThreadPoolwarm": "LibertyDTThroughput",
+        "17dev-4way-LargeThreadPool": "LibertyDTThroughput",
+        "17dev-4way-LargeThreadPoolwarm": "LibertyDTThroughput"
+    },
+    "SPECjbb2015GMR": {
+        "multi_2grp_gencon": "SPECjbb2015"
+    },
+    "WebSphereStartupEE": { 
+        "WAS855": "WebSphereStartup",
+        "WAS9000": "WebSphereStartup"
+    },
+    "DayTrader30": {
+        "WAS855gen1GB_4way": "DayTrader30",
+        "WAS855gen1GB_4waywarm": "DayTrader30",
+        "WAS9000gen1GB_4way": "DayTrader30",
+        "WAS9000gen1GB_4waywarm": "DayTrader30"
+    },
+    "SOABBenh_10k10k": {
+        "WAS855gen1GB_4way": "SOABENCH"
+    },
+    "Crypto-BB-v3":{
+        "default": "Crypto-BB-v3",
+        "IBMJCEPlus": "Crypto-BB-v3"
+    },
+    "WebSphereFootprint": {
+        "WAS855": "WebSphereFootprint",
+        "WAS9000": "WebSphereFootprint"
+    },
+    "WebSphereFootprintEE": {
+        "WAS855": "WebSphereFootprint",
+        "WAS9000": "WebSphereFootprint" 
+    },
+    "WebSphereStartup": { 
+        "WAS855": "WebSphereStartup",
+        "WAS855_2way": "WebSphereStartup",
+        "WAS855_NoSec": "WebSphereStartup",
+        "WAS855_NoSecXint": "WebSphereStartup",
+        "WAS855_Xint": "WebSphereStartup",
+        "WAS855_2way": "WebSphereStartup",
+        "WAS9000": "WebSphereStartup"
+    },
```

Comment:
The above values look redundant to me. I think we need to clean up this file. It can be done in another PR.